source/Tests/Playnite.Tests/FileSystemTests.cs

```diff
@@ -68,5 +68,27 @@ public void CheckSumTests()
             StringAssert.AreEqualIgnoringCase("46fcb37aa8e69b4ead0d702fd459299d", FileSystem.GetMD5(testFile));
             StringAssert.AreEqualIgnoringCase("D8B22F5D", FileSystem.GetCRC32(testFile));
         }
+
+        [Test]
+        public void DirectorySizeScanTest()
+        {
+            var testDir = Path.Combine(PlayniteTests.TempPath, "TestDir");

```

Comment:
You can use `using (var tempPath = TempDirectory.Create())` to create temp test directory that will be empty and specific for your test, so you don't have to make sure you have any files cleared from previous tests etc. See `ScannerTests` for how to use it.

Reply:
Done in https://github.com/JosefNemec/Playnite/pull/3059/commits/a00847bdedb4c7d97f941671f6b3069d8a14698f